routers/web/repo/issue.go

```diff
@@ -327,6 +327,20 @@ func issues(ctx *context.Context, milestoneID, projectID int64, isPullOption uti
 		}
 		return 0
 	}
+
+	if ctx.Repo.CanWriteIssuesOrPulls(ctx.Params(":type") == "pulls") {
```

Comment:
That query param was already present and wasn't added in this PR, right?

Reply:
Yes, it was already available (and is used in `func Issues` few lines below)